.vscode/settings.json

```diff
@@ -13,8 +15,8 @@
         "./bazel-out/_coverage/_coverage_report.dat"
     ],
     "lcov.path": [
-        "./.bazel-out/_coverage/_coverage_report.dat"
+        "./bazel-out/_coverage/_coverage_report.dat"
```

Comment:
Do you have a sample .dat file for us to look at?

Reply:
This is the same file you'd expect from `blaze coverage` in lcov format. https://bazel.build/configure/coverage